packages/apps/frontera/src/routes/src/components/CommandMenu/commands/contacts/ChangeOrAddJobRoles.tsx

```diff
@@ -76,6 +76,13 @@ export const ChangeOrAddJobRoles = observer(() => {
           if (e.key === ' ') {
             e.stopPropagation();
           }
+
+          if (e.metaKey && e.key === 'Enter') {
+            store.ui.commandMenu.setOpen(false);
+          } else {
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            handleSelect(search as any);
```

Comment:
can we handle the type here differently? I assume search is a string or undefined, let's add a typeguard but let's avoid 'as any' whenever possible